exercises/go-1-b/.meta/example_struct.go

```diff
@@ -0,0 +1,83 @@
+package go1b
+
+const maxScore = 21
+
+type Card int
```

Comment:
Why export this? Same with all the constants. They are not usable in any valueable context.